src/api.tsx

```diff
@@ -118,6 +118,17 @@ const authenticate = (credentials: {
     return (await resp.json()).jwt
   })
 
+const register = (credentials: {
```

Comment:
Do you want to name this `registerUser`, to be consistent with the other methods?